src/Apps/Order/Components/BankAccountPicker.tsx

```diff
@@ -16,8 +17,11 @@ interface Props {
   me: BankAccountPicker_me
   bankAccountHasInsufficientFunds: boolean
   setBankAccountHasInsufficientFunds: (arg: boolean) => void
-  onBankAccountContinue: () => void
   setIsSavingPayment: (arg: boolean) => void
+  setBalanceCheckComplete: (arg: boolean) => void
+  setSelectedBankAccountId: (arg: string) => void
+  bankAccountSelection: BankAccountSelection
+  setBankAccountSelection: (arg: BankAccountSelection) => void
```

Comment:
@AlicanAkyuz - this is non-blocking but worth noting, re these passed call-back functions:

The long-term React idiom for callback functions passed into a component is `onSomethingFoo` and local `useState` functions are `setSomethingFoo`

```
<MyComponent onBankBalanceCheckComplete={handleBankBalanceCheckComplete} />
```

A future dev reading this code will see these passed props and think that the state is locally set in this component, only to realize that what's actually being invoked is a callback function passed in as a prop. 

I'll comment further below to help explain the thinking...